src/dltParser.ts

```diff
@@ -93,6 +93,7 @@ export class DltMsg {
     readonly timeAsNumber: number; // time in ms. Date uses more memory!
     get timeAsDate(): Date { return new Date(this.timeAsNumber); }
 
+    readonly payloadArgSeparator: string;
```

Comment:
this is increasing the memory needs per msg. Should be avoided.

Reply:
Shouldn't be a big issue, if the legacyApproach is taken will be unnecessary.

Reply:
Removed the complete implementation and hard coded the space after each payload arg.

Reply:
Which might now have been the smartest idea. I guess we need to talk more about the behaviour of the legacyPayloadDecoding feature. 
How to achieve a similar behaviour w/o increasing the memory/processing overhead?